docs/visual-basic/language-reference/statements/on-error-statement.md

```diff
@@ -24,94 +24,92 @@ helpviewer_keywords:
 ms.assetid: ff947930-fb84-40cf-bd66-1ea219561d5c
 ---
 # On Error Statement (Visual Basic)
-Enables an error-handling routine and specifies the location of the routine within a procedure; can also be used to disable an error-handling routine.  
-  
- Without error handling, any run-time error that occurs is fatal: an error message is displayed, and execution stops.  
-  
- Whenever possible, we suggest you use structured exception handling in your code, rather than using unstructured exception handling and the `On Error` statement. For more information, see [Try...Catch...Finally Statement](../../../visual-basic/language-reference/statements/try-catch-finally-statement.md).  
-  
+Enables an error-handling routine and specifies the location of the routine within a procedure; can also be used to disable an error-handling routine.
+
+ Without error handling, any run-time error that occurs is fatal: an error message is displayed, and execution stops.
+
```

Comment:
Suggest a one sentence paragraph here something like the following. This is a first cut, happy to see it rewritten or you can just copy it into your PR, but I'd like to set the two approaches somewhat in relation here, and mention the encouragement to structured early. 

The `On Error` statement is part of unstructured error handling and can be used instead of [structured exception handling](../../../visual-basic/language-reference/statements/try-catch-finally-statement.md). Structured exception handling is built into .NET, so is generally more efficient.